brianlong/www.validators.app
# 184544324 filter by jito

#### What's this PR do?
- adds script that connects with the JITO api and assigns correct flag to validators in our db
- adds filter on /validators page displaying only jito validators

#### How should this be manually tested?
- run migrations
- run tests
- run script `rails r script/update_jito_validators.rb` and check if the flag is updated
- see if the filter works properly on /validators page

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184544324)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
